File: app/src/main/java/md/fusionworks/lifehack/ui/widget/ClickFixNestedScrollView.java
package md.fusionworks.lifehack.ui.widget;

import android.content.Context;
import android.support.v4.widget.NestedScrollView;
import android.util.AttributeSet;
import android.view.MotionEvent;

public class ClickFixNestedScrollView extends NestedScrollView {

    public ClickFixNestedScrollView(Context context, AttributeSet attrs) {
        super(context, attrs);
    }

    @Override
    public boolean onInterceptTouchEvent(MotionEvent ev) {
        if (ev.getAction() == MotionEvent.ACTION_DOWN) {
            // Explicitly call computeScroll() to make the Scroller compute itself
            computeScroll();
        }
        return super.onInterceptTouchEvent(ev);
    }
}

File: app/src/main/java/md/fusionworks/lifehack/widget/ClickFixNestedScrollView.kt
package md.fusionworks.lifehack.widget

import android.content.Context
import android.support.v4.widget.NestedScrollView
import android.util.AttributeSet
import android.view.MotionEvent

class ClickFixNestedScrollView(context: Context, attrs: AttributeSet) : NestedScrollView(context,
    attrs) {

  override fun onInterceptTouchEvent(ev: MotionEvent): Boolean {
    if (ev.action == MotionEvent.ACTION_DOWN) {
      // Explicitly call computeScroll() to make the Scroller compute itself
      computeScroll()
    }
    return super.onInterceptTouchEvent(ev)
  }
}